### PR TITLE
(maint) Add CONTRIBUTING and Maintenance doc sections

### PR DESCRIPTION
**Pull request by parisiale**

### CODE REVIEW

**Comment by parisiale** [comment]
@kylog @MikaelSmith @mruzicka , I'll update the other repos once we agree on this.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# How to contribute
+
+Third-party patches are essential for keeping Puppet Labs open-source projects
```
Comment (mruzicka) [inline CONTRIBUTING.md]: Puppet Labs -> Puppet ?

File: README.md [hunk 2]
```diff
@@ -346,6 +346,16 @@ Don't become a daemon and execute on foreground on the associated terminal.
 
 The path of the PID file; the default is */var/run/puppetlabs/pxp-agent.pid*
 
+## Maintenance
+
+Maintainers: Alessandro Parisi <alessandro@puppet.com>, Michael Smith
+<michael.smith@puppet.com>, Michal Ruzicka <michal.ruzicka@puppetlabs.com>.
```
Comment (mruzicka) [inline README.md]: uh, oh: `<michal.ruzicka@puppetlabs.com>` -> `<michal.ruzicka@puppet.com>`
sorry for that, didn't notice it in the HipChat

File: CONTRIBUTING.md (hunk 1 above)
Comment (parisiale) [inline CONTRIBUTING.md]: 👍

File: README.md (hunk 2 above)
Comment (parisiale) [inline README.md]: 👍

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# How to contribute
+
+Third-party patches are essential for keeping Puppet open-source projects
+great. We want to keep it as easy as possible to contribute changes that
+allow you to get the most out of our projects. There are a few guidelines
+that we need contributors to follow so that we can have a chance of keeping on
+top of things.  For more info, see our canonical guide to contributing:
+
+[https://github.com/puppetlabs/puppet/blob/master/CONTRIBUTING.md](https://github.com/puppetlabs/puppet/blob/master/CONTRIBUTING.md)
```
Comment (MikaelSmith) [inline CONTRIBUTING.md]: Someday we should probably put that on puppet.com, or somewhere that's not the Puppet repo.

**Pull request merged**